source/type/vehicle_journey.cpp

```diff
@@ -227,18 +229,27 @@ ValidityPattern VehicleJourney::get_vp_for_section(const std::set<StopPoint*>& s
     return vp_for_section;
 }
 
-std::set<StopPoint*> VehicleJourney::get_sections_stop_points(const StopArea* start_stop,
-                                                              const StopArea* end_stop) const {
+std::set<uint16_t> VehicleJourney::get_sections_ranks(const StopArea* start_stop, const StopArea* end_stop) const {
```

Comment:
As there are StopTime, StopPoint, StopArea used after, for clarity (maybe other var need renames):
```suggestion
std::set<uint16_t> VehicleJourney::get_sections_ranks(const StopArea* start_sa, const StopArea* end_sa) const {
```

Reply:
Thanks for the suggestion